Formula/pypy.rb

```diff
@@ -55,7 +56,23 @@ class Pypy < Formula
     sha256 "6773934e5f5fc3eaa8c5a44949b5b924fc122daa0a8aa9f80c835b4ca2a543fc"
   end
 
+  # Build fixes:
+  # - Disable Linux `tcl-tk` detection since the build script only searches system paths.
+  # - Add arg `--no-make-portable` so that the default portable macOS build can be disabled.
+  patch :DATA
+
   def install
+    # The `tcl-tk` library paths are hardcoded and need to be modified for non-/usr/local prefix
+    inreplace "lib_pypy/_tkinter/tklib_build.py", "/usr/local/opt/tcl-tk/", Formula["tcl-tk"].opt_prefix/""
+
+    # Help find Homebrew libraries
+    ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["openssl@1.1"].opt_lib/"pkgconfig"
```

Comment:
This should already be in `PKG_CONIG_PATH`. Superenv does this for keg-only dependencies and adds `$HOMEBREW_PREFIX/lib/pkgconfig` for non-keg-only ones.

Reply:
Yeah, I saw it was probably a no-op when looking at logs. Right now, just throwing everything at the build to see anything works.

The only thing I saw was `pkg-config` should have helped pick up Homebrew OpenSSL, but it doesn't seem to work. Or, it is statically linking it.

If I can't figure it out, I'll just leave the libs as is since that is how we've been distributing it.

Reply:
Local build attempt still built OpenSSL 1.1.1k. No idea how to avoid this, so will probably just leave for now and let someone else look into in a separate PR. Main focus items are fixed.
```
/private/tmp/pypy-20210813-95966-1txhgk9/pypy2.7-v7.3.5-src/lib-python/2.7/hashlib.py:146: RuntimeWarning: The _hashlib module is not available, falling back to a much slower implementation (No module named _pypy_openssl)
  RuntimeWarning)
* _ctypes/_ctypes_build.py
* _ssl_build.py
fetching archive http://distfiles.macports.org/openssl/openssl-1.1.1k.tar.gz
unpacking archive /private/tmp/pypy-archives/openssl-1.1.1k.tar.gz
running ./config --prefix=/usr no-shared in /private/tmp/pypy-20210813-95966-1txhgk9/pypy2.7-v7.3.5-src/lib_pypy/pypy_tools/dest/openssl-1.1.1k
running make -s -j 8 in /private/tmp/pypy-20210813-95966-1txhgk9/pypy2.7-v7.3.5-src/lib_pypy/pypy_tools/dest/openssl-1.1.1k
running make install DESTDIR=/private/tmp/pypy-20210813-95966-1txhgk9/pypy2.7-v7.3.5-src/lib_pypy/pypy_tools/dest/ in /private/tmp/pypy-20210813-95966-1txhgk9/pypy2.7-v7.3.5-src/lib_pypy/pypy_tools/dest/openssl-1.1.1k
```